Formula/dynare.rb

```diff
@@ -0,0 +1,79 @@
+class Dynare < Formula
+  desc "Platform for economic models, particularly DSGE and OLG models"
+  homepage "https://www.dynare.org/"
+  url "https://www.dynare.org/release/source/dynare-4.5.3.tar.xz"
+  sha256 "01434f6d3ceaff1891dc771f6a5b39caee787b2ffa1875e5a4e8c673e32ff3d7"
+
+  head do
+    url "https://github.com/DynareTeam/dynare.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "bison" => :build
```

Comment:
macOS ships both `bison` and `flex`; it's worth trying removing these dependencies locally and seeing if the `HEAD` build still works.

Reply:
dies without them

```
bison -o DynareBison.cc DynareBison.yy
DynareBison.yy:21.10-14: require bison 2.5, but have 2.3
bison -o DynareBison.cc DynareBison.yy
DynareBison.yy:21.10-14: require bison 2.5, but have 2.3
make[1]: *** [stack.hh] Error 63
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [DynareBison.hh] Error 63
DynareBison.yy:21.10-14: require bison 2.5, but have 2.3
make[1]: *** [position.hh] Error 63
DynareBison.yy:21.10-14: require bison 2.5, but have 2.3
make[1]: *** [location.hh] Error 63
DynareBison.yy:21.10-14: require bison 2.5, but have 2.3
make[1]: *** [DynareBison.cc] Error 63
make: *** [install-recursive] Error 1
```
and
```
flex -o DynareFlex.cc DynareFlex.ll
cp /usr/bin/../include/FlexLexer.h . || test -f ./FlexLexer.h
cp: /usr/bin/../include/FlexLexer.h: No such file or directory
make[1]: *** [FlexLexer.h] Error 1
make[1]: *** Waiting for unfinished jobs....
cp /usr/bin/../include/FlexLexer.h . || test -f ./FlexLexer.h
cp: /usr/bin/../include/FlexLexer.h: No such file or directory
make[1]: *** [DynareFlex.cc] Error 1
make: *** [install-recursive] Error 1
```
```